docs/users/cli-usage.md

```diff
@@ -5,12 +5,12 @@ Type `ddev` or `ddev -h`in a terminal windows to see the available ddev commands
 
 ## Quickstart Guides
 
-You can start using ddev by running just a few commands. Below are quickstart instructions WordPress, Drupal 7, Drupal 8, and TYPO3 CMS.
+These are quickstart instructions WordPress, Drupal 7, Drupal 8, TYPO3, and Backdrop.
 
-**Prerequisites:** If you do not have ddev already on your machine, please follow the [installation instructions](../index.md#installation) before beginning the quickstart tutorials.  You'll need *docker* and *docker-compose* to use ddev.
+**Prerequisites:** Before you start, [check the system requirements](../index.md#system-requirements) and follow the [installation instructions](../index.md#installation).
 
 ### WordPress Quickstart
-To get started using ddev with a WordPress project, simply clone the project's repository and checkout its directory.
+To get started using ddev with a WordPress project clone the project's repository and checkout its directory.
```

Comment:
I do think it needs a comma after project.

Reply:
OK!